synthtool/gcp/common.py

```diff
@@ -93,10 +93,10 @@ def _load_generic_metadata(self, metadata: Dict):
         self._load_samples(metadata)
         self._load_partials(metadata)
 
-        metadata["repo"] = {}
-        if os.path.exists("./.repo-metadata.json"):
-            with open("./.repo-metadata.json") as f:
-                metadata["repo"] = json.load(f)
+        if "repo" not in metadata:
+            repo_metadata = load_repo_metadata()
+            if repo_metadata:
+                metadata["repo"] = repo_metadata
```

Comment:
I'm really excited to see us converging on `.repo-metadata.json` for this type of stuff. Good work @busunkim96, patience goes a long way to adoption.

Reply:
Added a comment - we may already be loading the repo metadata elsewhere so no need to do it again and overwrite the values.